package.json

```diff
@@ -49,10 +49,11 @@
   "scripts": {
     "test": "istanbul cover -x '**/*.test.js' node_modules/mocha/bin/_mocha tests/all -- -R spec --recursive",
     "lint": "eslint .",
-    "simple-integration-test": "mocha tests/integration/simple-integration-test",
-    "complex-integration-test": "mocha tests/integration/all"
+    "simple-integration-test": "ava tests/integration/simple-integration-test.js",
+    "complex-integration-test": "ava -c 10 tests/integration/general/**/tests.js tests/integration/aws/**/tests.js"
```

Comment:
any reason for splitting general and aws tests here? If we add another folder it wouldn't be picked up in this case.